integration/tests/cook/test_cli.py

```diff
@@ -1606,6 +1614,10 @@ def test_cat_bogus_uuid(self):
         self.assertEqual(1, cp.returncode, cp.stdout)
         self.assertIn('No matching data found', cli.decode(cp.stderr))
 
+    @unittest.skipIf(util.has_ephemeral_hosts(), 'If the cluster under test has ephemeral hosts, then it is generally '
+                                                 'a bad idea to use HOSTNAME EQUALS constraints, because it can cause '
+                                                 'the process responsible for launching hosts to launch hosts that '
+                                                 'never get used')
     def test_cat_job_with_no_instances(self):
         raw_job = {'command': 'ls', 'constraints': [['HOSTNAME', 'EQUALS', 'will not get scheduled']]}
```

Comment:
Is it possible to use a different constraint that causes jobs to not be scheduled and avoid the issue of launching hosts that never get used?

Reply:
The CLI tests don't currently support docker jobs anyway, so I removed the `skipIf` until we address that in a separate PR